Sources/RInAppMessaging/Extensions/UIApplication+IAM.swift

```diff
@@ -35,4 +35,30 @@ extension UIApplication {
         // Only iOS 13+ is supported
         return nil
     }
+    
+    private static let notificationSettingsURLString: String? = {
+        if #available(iOS 16, *) {
+            return UIApplication.openNotificationSettingsURLString
+        }
+        
+        if #available(iOS 15.4, *) {
+            return UIApplicationOpenNotificationSettingsURLString
+        }
+        
+        if #available(iOS 8.0, *) {
+            return UIApplication.openSettingsURLString
+        }
+        
+        return nil
+    }()
+    
+    private static let appNotificationSettingsURL = URL(
+        string: notificationSettingsURLString ?? ""
+    )
+    
+    func openAppNotificationSettings() -> Void {
+        guard let url = UIApplication.appNotificationSettingsURL else { return }
+        self.open(url)
```

Comment:
Better to check `canOpenURL(url)` before try to open the any URL which would avoid crash